packages/orbit-components/src/Tile/components/TileWrapper/index.tsx

```diff
@@ -94,31 +32,37 @@ const TileWrapper = ({
   ariaControls,
   id,
   htmlTitle,
-}: React.PropsWithChildren<Props>) => (
-  <StyledTileWrapper
-    data-test={dataTest}
-    onClick={onClick}
-    onKeyDown={onKeyDown}
-    as={as}
-    tabIndex={tabIndex}
-    role={role}
-    aria-expanded={ariaExpanded}
-    aria-controls={ariaControls}
-    id={id}
-  >
-    {href ? (
-      <StyledTileAnchor
-        target={href && external ? "_blank" : undefined}
-        rel={href && external ? "noopener" : undefined}
-        href={href || undefined}
-        title={htmlTitle}
-      >
-        {children}
-      </StyledTileAnchor>
-    ) : (
-      children
-    )}
-  </StyledTileWrapper>
-);
+}: React.PropsWithChildren<Props>) => {
+  const WrapperComponent = (as ?? "div") as React.ElementType;
+
+  return (
+    <WrapperComponent
+      className="orbit-tile-wrapper hover:[&_.orbit-tile-header-icon-right]:text-ink-light-hover font-base text-ink-dark shadow-action duration-fast hover:shadow-action-active rounded-large bg-white-normal de:rounded-normal box-border block w-full no-underline transition-shadow ease-in-out"
```

Comment:
`hover:[&_.orbit-tile-header-icon-right]:text-ink-light-hover` This class doesn't seem to be having any effect, but it should. It's causing visual differences on hover when compared to master. Check below how the icon color doesn't change, as it should.

Master:
![master](https://github.com/kiwicom/orbit/assets/6265045/1685dadf-4884-40d5-a7e2-67d438c650f2)


Here:
![branch](https://github.com/kiwicom/orbit/assets/6265045/3f7bb5ac-a9ef-4519-93ef-9dd62cf41bfd)